### ISSUE TITLE
Display [messages] that contain options or text inputs in replays

### ISSUE THREAD

**Issue by DeFender1031** [wesnoth/wesnoth#1834]
Currently, replays skip messages which contained interactive components. This can make the dialogue that involved them hard to follow. A simple way to include them would be to show a non-interactive message which has the selected option highlighted in some fashion, and the rest of them dimmed or grayed out in some fashion in the case of options, or a non-editable text input which contains the text as it was originally input in the case of a text input. Either of these cases should still otherwise work the same way as other messages (dismissed by clicking or hitting space or enter, skipped if the user hit esc on a previous message, etc.)

**Comment by CelticMinstrel** [comment]
Another option would be to compose a non-interactive message from the concatenation of the base message, the option that had been selected, and whatever text had been entered (probably including the textbox label if that exists, I can't remember).
